### PR TITLE
Fix nightly staging docker

### PR DESCRIPTION
**Pull request by FlappiTomic**
Signed-off-by: Tom Zhang <14881754+FlappiTomic@users.noreply.github.com>

Please check if your PR fulfills the following requirements. This is simply a reminder of what we are going to look for before merging your PR. If you don't know all of this information when you create this PR, don't worry. You can edit this template as you're working on it.

- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Necessary documentation (if appropriate) have been added / updated
- [x] DCO signoffs have been added to all commits, including this PR

#### PR type
What type of changes does your PR introduce to Zowe? _Put an `x` in the box that applies to this PR. If you're unsure about any of them, don't hesitate to ask._ 
- [x] Bugfix 
- [ ] Feature 
- [ ] Other... Please describe:

#### Relevant issues

Fixes 

#### Changes proposed in this PR

- 

#### Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

#### Does this PR do something the person installing Zowe should know about?

---
* Affected function: _general area of interest_ *

---
* Description: _1 line description_ *

---
* Part: _name of customizable file involved_  *

---
_multi-line description_

#### Is there a related doc issue or Pull Request? 

Doc issue/PR number:

### CODE REVIEW

**Comment by FlappiTomic** [comment]
@jackjia-ibm I think this is the fix for this issue...

File: playbooks/roles/docker/tasks/docker-start.yml [hunk 1]
```diff
@@ -9,6 +9,7 @@
       --env ZWED_agent_https_port={{zowe_zss_port}} \
       --env ZOWE_ZLUX_TELNET_PORT={{zowe_zlux_terminal_telnet_port}} \
       --env ZOWE_ZLUX_SECURITY_TYPE={{ zowe_zlux_terminal_telnet_security_type }} \
+      --env ZOWE_ZLUX_SSH_HOST=zzow01.zowe.marist.cloud \
```
Comment (jackjia-ibm) [inline playbooks/roles/docker/tasks/docker-start.yml:12]: Should use `zowe_zos_host` variable.

**Review by jackjia-ibm — CHANGES_REQUESTED** [review]
Let's leave this PR open for a while and see if we can find more on the root cause. It doesn't make sense why SSH `localhost` works for conv build test but failed for docker test.

**Comment by 1000TurquoisePogs** [comment]
I'm confused how this happened, since the code for `ZOWE_ZLUX_SSH_HOST` hasn't changed in the year 2021.

File: playbooks/roles/docker/tasks/docker-start.yml (hunk 1 above)
Comment (FlappiTomic) [reply playbooks/roles/docker/tasks/docker-start.yml]: Changed

**Pull request merged**